hack/tools/tilt-prepare/main.go

```diff
@@ -72,8 +75,23 @@ var (
 	kustomizeBuildsFlag  = pflag.StringSlice("kustomize-builds", []string{}, "list of kustomize build to be run; each value should be in the form name:path")
 	providersBuildsFlag  = pflag.StringSlice("providers", []string{}, "list of providers to be installed; each value should be in the form name:path[:debug]")
 	allowK8SContextsFlag = pflag.StringSlice("allow-k8s-contexts", []string{}, "Specifies that Tilt is allowed to run against the specified k8s context name; Kind is automatically allowed")
+	tiltSettingsFileFlag = pflag.String("tilt-settings-file", "", "Path to a tilt-settings.(json|yaml) file")
 )
 
+type debugConfig struct {
+	Continue     *bool `json:"continue"`
+	Port         int   `json:"port"`
+	ProfilerPort int   `json:"profiler_port"`
+	MetricsPort  int   `json:"metrics_port"`
```

Comment:
I would suggest making them all pointers to signal that they are optional, and then we can just check for nil to detect if they are set or not